src/appMain/main.cc

```diff
@@ -101,7 +101,6 @@ int32_t main(int32_t argc, char** argv) {
   } else {
     profile::Profile::instance()->config_file_name("smartDeviceLink.ini");
   }
-
   PLATFORM_INIT(argc, argv);
```

Comment:
This should be almost the first line in the main. There is QtCoreApplication creation behind